testing/snapshot-test.md

```diff
@@ -0,0 +1,20 @@
+# Snapshot test
+
+## Introduction
+Snapshot tests are used to test visual regression of presentational components and for reducers in order to avoid deep comparison boilerplate code
+
+## Dos
+- The visual component output has to be relatively small.  
+- Focus to shallow snapshot comparison.
+- In cases where snapshot output is not readable use enzyme-to-json utility.
+- Create snapshot for every presentational component.  
```

Comment:
I don't agree with this. We dont really need snapshots for _every_ presentational component. Often, it depends on the use case. There are cases when we don't need snapshots but  testing based on 
- if specific elements are present
- state changes
- event handlers etc